parsers/IN_EA.py

```diff
@@ -7,7 +7,7 @@
 
 from electricitymap.contrib.config import ZoneKey
 from electricitymap.contrib.lib.models.event_lists import ExchangeList
-from electricitymap.contrib.parsers.lib.config import refetch_frequency
```

Comment:
I assume you are on windows?
I had the same issues so this change is perfectly fine.